linux-cachyos-rc/PKGBUILD

```diff
@@ -163,7 +163,7 @@ _is_clang_kernel() {
 
 if [[ "$_use_llvm_lto" = "thin" || "$_use_llvm_lto" = "full" ]] && [ "$_use_lto_suffix" = "y"  ]; then
     _pkgsuffix="cachyos-rc-lto"
-elif [ "$_use_llvm_lto" = "none" ]  && [ -z "$_use_kcfi" ] && [ "$_use_gcc_suffix" = "y" ]; then
+elif [ "$_use_llvm_lto" = "none" ]  && [ -z "$_use_kcfi" ] && [ "$_use_gcc_suffix" = "y" ] && [ -z "$_autofdo" ]; then
```

Comment:
```suggestion
elif ! _is_clang_kernel && [ "$_use_gcc_suffix" = "y" ]; then
```

Reply:
The same can be used in the condition above

Reply:
For the first condition it doesn't seem "correct" to mark the kernel as LTO when LTO is not enabled but the other clang features are (kCFI, autofdo). I guess what we can do is make LTO a hard dependency for building with clang to avoid any fallacies. Thoughts? @ptr1337 @ventureoo

Reply:
> For the first condition it doesn't seem "correct" to mark the kernel as LTO when LTO is not enabled but the other clang features are (kCFI, autofdo).

Hmm... Yeah, I overlooked that a bit, although I'm not sure who has desire to build kernel via Clang without LTO, but it makes sense.